src/components/contextless-ios-view-controller-toggle/contextless-ios-view-controller-toggle.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ToggleableCodeBlock from '../toggleable-code-block/toggleable-code-block';
+import { highlightSwift } from '../../helpers/highlight-swift';
+import { highlightObjectivec } from '../../helpers/highlight-objectivec';
+
+export default class ContextlessIosViewControllerToggle extends React.Component {
+  static defaultProps = {
+    limitHeight: true
+  };
+
+  render() {
+    const { id, context, objectiveC, swift, limitHeight } = this.props;
+
+    let selectedCode = '';
+    if (context) {
+      selectedCode =
+        this.props.context.preferredLanguage.ios === 'objectiveC'
+          ? objectiveC
+          : swift;
+    }
+
+    const snippetProps = {
+      copyRanges: this.props.copyRanges || undefined,
+      filename: this.props.filename || undefined,
+      link: this.props.link || undefined,
+      options:
+        objectiveC && swift
+          ? [
+              {
+                label: 'Swift',
+                language: 'swift',
+                preferredLanguage:
+                  this.props.context.preferredLanguage.ios === 'swift'
+              },
+              {
+                label: 'Objective-C',
+                language: 'objectiveC',
+                preferredLanguage:
+                  this.props.context.preferredLanguage.ios === 'objectiveC'
+              }
+            ]
+          : undefined
+    };
+
+    return (
+      <div className="my24">
```

Comment:
Like Android, you'll also want the prose class here, too.

```suggestion
      <div className="my24 prose">
```